stroeer/core/v1/article.proto

```diff
@@ -847,15 +847,16 @@ Article metadata like publication state and technical timestamps.
+| `state`                  | [`State`][state]  | State of the article in the content management system. See [`enum State`](#enum-state)                                                                                                                                                                                                                                  |
+| `start_time`             | [`Timestamp`][ts] | Manually set editorial timestamp (_Gültig von_) at which the article is valid to deliver on digital platforms in seconds of UTC time since Unix epoch.                                                                                                                                                                  |
+| `end_time`               | [`Timestamp`][ts] | Manually set editorial date (_Gültig bis_) till the article is valid to deliver on digital platforms in seconds of UTC time since Unix epoch.                                                                                                                                                                           |
+| `publish_time`           | [`Timestamp`][ts] | Editorial date (_Publikationsdatum_) of the first publication of the article in seconds of UTC time since Unix epoch. This date will be set automatically by the content management system.                                                                                                                             |
+| `update_time`            | [`Timestamp`][ts] | Editorial date (_Aktualisierungsdatum_) at which the article was updated in seconds of UTC time since Unix epoch. On first publication this timestamp matches `publish_time`. Afterwards it's either updated manually in the content management system or automatically if the article content changed *significantly*. |
+| `transformation_time`    | [`Timestamp`][ts] | Technical timestamp at which the article was transformed in the API layer in seconds of UTC time since Unix epoch.                                                                                                                                                                                                      |
+| `transformation_errors`  | `int64`           | Number of errors occurred while fetching and/or transforming optional article components (e.g. `embeds` or nested `documents`) to an `article` message.                                                                                                                                                                 |
+| `last_modification_time` | [`Timestamp`][ts] | Technical timestamp at which the article was published regardless of the amount and significance of the change.                                                                                                                                                                                                        |
```

Comment:
We should align the terms _date_ and _timestamp_ (with and w/o UTC explanation) in the docs here, otherwise 👍